src/XR/motionController/webXRGenericHandController.ts

```diff
@@ -0,0 +1,138 @@
+import { IMotionControllerLayoutMap, IMinimalMotionControllerObject, MotionControllerHandedness, WebXRAbstractMotionController } from "./webXRAbstractMotionController";
+import { Scene } from "../../scene";
+import { AbstractMesh } from "../../Meshes/abstractMesh";
+import { Mesh } from "../../Meshes/mesh";
+import { Quaternion } from "../../Maths/math.vector";
+import { WebXRMotionControllerManager } from "./webXRMotionControllerManager";
+
+/**
+ * The motion controller class for the standard HTC-Vive controllers
+ */
+export class WebXRGenericHandController extends WebXRAbstractMotionController {
+
+    public profileId = "generic-hand-select-grasp";
+
+    /**
+     * Create a new Vive motion controller object

```

Comment:
nit - documentation. WOuld be great to document the class correctly for the typedoc page

Reply:
Oh jeez I forgot to update the comments